literature/horswell_2004_a_10970/horswell_2004_a_10970_p2_1a_solid.yaml

```diff
@@ -0,0 +1,102 @@
+
+electrochemical system:
+    version: 1
+    instrumentation:
+        - type:
+          supplier: 
+        - type:
+          model:
+          supplier:
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: NaF # can be trivia name, sum formula, etc
+              concentration:
+                  value: 0.09
+                  unit: M #[M, mM, µM, g kg-1, ...]
+              type: salt #
+              source:
+                  supplier:
+                  LOT: 
+            - name: water
+              type: solvent
+              proportion:
+                  value:
+                  unit:
+              source:
+                  refinement:
+                  quality:
+                  total organic carbon: 
+                      value:
+                      unit:
+            - name: 
+              type:
+              proportion:
+                  value: 
+                  unit:
+            - name: NaOH
+              type: base
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value: 0.01
+                  unit: M #[M, mM, µM, g kg-1, ...] 
+        ph: 
+            value:
+            uncertainty:
+        temperature: # provide the temperature as it is given in the article. Assume room temperature if nothing is provided
+            value: 298.15 
+            unit: K # [K, C, F]
+    electrodes:
+        configuration: 3 # number of electrodes
+        reference electrode:
+            type: Hg/HgO/0.1 M NaOH # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: ring
+        working electrode:
+            material: Ag
+            crystallographic orientation: '110' # hkl, 100, 110, poly
+            source:
+                supplier:
+                LOT: 
+            shape: # optional
+                type: # bead/sphere
+                height:
+                    value:
+                    unit:
+                diameter:
+                    value:
+                    unit:
+            geometric electrolyte contact area: 
+                value:
+                unit:
+            preparation procedure: Ag surface prepared with NaCN/H2O2 etching and annealing in H2 flame under Ar # Short description of sample preparation. This can be very useful. :)

```

Comment:
```suggestion
            preparation procedure: Ag surface prepared with NaCN/H2O2 etching and annealing in H2 flame under Ar. # Short description of sample preparation. This can be very useful. :)

```